src/Renci.SshNet/ScpClient.NET.cs

```diff
@@ -15,6 +16,14 @@ public partial class ScpClient
         private static readonly Regex TimestampRe = new Regex(@"T(?<mtime>\d+) 0 (?<atime>\d+) 0");
 
         /// <summary>
+        /// Transferring UNIX files to Windows may require special treatment.
+        /// Enabling this will remove the following special characters from filenames when copying whole directories:
+        /// " &lt; &gt; | : * ? \ /
+        /// </summary>
+        public bool UnixToWindows { get; set; }
```

Comment:
No, not particularly. It was just my way of making sure any users knew that the filenames would be changed because they had to specifically request it, with a log and summary note it shouldn't be necessary.

I'd be torn on whether to always apply the change. Personally I don't see why there'd ever be a good reason to keep special characters that can break things in cross-system setups. But then maintaining original file names where possible might be preferred for some people.

Should be possible to just apply for Windows environments.

Also that test file looks...interesting, I'll take a look in the next few days when I get a chance.